<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserCard</class>
 <widget class="QWidget" name="UserCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Номер:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="selfNumber">
          <property name="text">
           <string>0000</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Текущий статус:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="acceptCall">
          <property name="text">
           <string>Снять трубку</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dropCall">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Положить трубку</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Номер вызываемого:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="subscriberNumber">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="callButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Позвонить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Текст разговора:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="talkText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
